Misc/NEWS.d/next/C API/2021-06-15-22-25-00.bpo-44392.9gf_Qu.rst

```diff
@@ -0,0 +1,2 @@
+Fix the exported return type of ``Py_GenericAliasType`` for MSVC.  The
```

Comment:
It's not the *return* type, is it? Maybe there's more precise terminology but I think we can just say "type".